.github/workflows/source.yml

```
@@ -41,3 +41,30 @@ jobs:
         with:
           name: colcon-logs-${{ matrix.os }}
           path: ros_ws/log
+
+  ci_source_clang:
+    name: Foxy source job
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: false
+    steps:
+      - uses: ros-tooling/setup-ros@v0.1
+      - uses: ros-tooling/action-ros-ci@v0.1
```

```suggestion
      - uses: ros-tooling/setup-ros@v0.2
      - uses: ros-tooling/action-ros-ci@v0.2
```

oops, you'll also need to bump this to be able to use `colcon-defaults`.